src/TinyPNG/Extensions/ConvertExtensions.cs

```diff
@@ -33,7 +33,16 @@ public static async Task<TinyPngConvertResponse> Convert(this Task<TinyPngCompre
 
         TinyPngCompressResponse compressResponse = await result;
 
-        string requestBody = JsonSerializer.Serialize(new { convert = new { type = convertOperation } }, TinyPngClient._jsonOptions);
+        string requestBody = JsonSerializer.Serialize(new {
+            convert = new
+            {
+                type = convertOperation
+            },
+            transform = new
+            {
+                background = backgroundTransform
+            }
```

Comment:
Ah, we need to actually set `transform` to null, otherwise it will have an empty `transform` property

Reply:
Done?

Reply:
Thanks for persisting with this !